scripts/Tools/Makefile

```diff
@@ -898,8 +899,10 @@ $(PIOLIB) : $(MPISERIAL) $(GPTLLIB)
 
 $(CSMSHARELIB):  $(MCTLIBS) $(PIOLIB) $(GPTLLIB)
 
-ifneq ($(MODEL),csm_share)
+ifneq ($(findstring csm_share,$(MODEL)),csm_share)
   $(OBJS):  $(CSMSHARELIB)
```

Comment:
Your right, I've added CSMSHARELIB_CPL7 to the dependency for the rebuild.  The original reason I changed the logic here was csm_share_cpl7 was throwing a copy error because there was there was no *.h files.